assets/sass/components/_applications.scss

```diff
@@ -0,0 +1,93 @@
+$statuses: (
+    'complete': #55ac3b,
+    'incomplete': #fd992b,
+    'empty': #7b7b7b,
+    'invalid': red
+);
+
+$statusColumnWidth: 160px;
+
+.app-summary__status {
+    display: flex;
+    justify-content: space-between;
+    border-left: 3px solid get-color('background', 'light-neutral');
+    padding-left: $spacingUnit / 2;
+
+    @each $state, $color in $statuses {
```

Comment:
Not a huge fan of generated class names, I find it reduces search-ability. I'd be inclined to write the classes out longhand.